src/test/java/io/confluent/idesidecar/restapi/messageviewer/RecordDeserializerTest.java

```diff
@@ -63,7 +63,7 @@ public class RecordDeserializerTest {
       null,
       null,
       null,
-      CONNECTION_1_ID,
+      new SchemaErrors.ConnectionId(CONNECTION_1_ID),
```

Comment:
I recently suggested think NOT exposing `SchemaErrors.ConnectionId` or `SchemaErrors.SchemaId` outside of `SchemaErrors`. If that's the case, why do we need (or want) the MessageViewerContext to have an instance of ``SchemaErrors.ConnectionId`?

I think it breaks encapsulation and abstraction if `MessageViewerContext` has a field of type `SchemaErrors.ConnectionId` just to hold onto a connection ID, as the connection ID is not at all related to SchemaErrors.